.github/workflows/build.yaml

```diff
@@ -0,0 +1,63 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    check:
+        runs-on: ubuntu-20.04
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Composer check
+              run: composer validate --strict --no-check-lock
```

Comment:
you still need the `shivammathur/setup-php@v2` step before that one, otherwise composer will not be available.